src/main/java/org/reaktivity/nukleus/http_cache/internal/proxy/cache/DefaultCache.java

```diff
@@ -110,27 +121,43 @@ public BufferPool getResponsePool()
     public DefaultCacheEntry get(
         int requestHash)
     {
-        return cachedEntries.get(requestHash);
+        return cachedEntriesByRequestHash.get(requestHash);
     }
 
     public boolean hasCacheEntry(
         int requestHash)
     {
-        return cachedEntries.containsKey(requestHash);
+        return cachedEntriesByRequestHash.containsKey(requestHash);
     }
 
     public DefaultCacheEntry supply(
-        int requestHash)
+        int requestHash,
+        String requestURL)
+    {
+        final int collectionHash = requestHashWithoutQuery(requestURL);
```

Comment:
Rename `collectionHash` to `requestHashWithoutQuery`.

Reply:
Any reason we cannot pass `requestHashWithoutQuery` instead of `requestURL` here?

Reply:
`requestHashWithoutQuery ` actually generates requestHashWithoutQuery. I modified the method name and variable name.